server/start.go

```diff
@@ -78,8 +78,8 @@ which accepts a path for the resulting pprof file.
 		FlagMinGasPrices, "",
 		"Minimum gas prices to accept for transactions; Any fee in a tx must meet this minimum (e.g. 0.01photino;0.0001stake)",
 	)
-	cmd.Flags().Bool(FlagSkipUpgrade, false, "Skip current software upgrade to continue the old binary")
-	cmd.Flags().Uint64(FlagHaltHeight, 0, "Block height at which to gracefully halt the chain and shutdown the node")
+	cmd.Flags().Uint64(FlagSkipUpgrade, 0, "Skip current software upgrade to continue the old binary")
+	cmd.Flags().Int64(FlagHaltHeight, -1, "Block height at which to gracefully halt the chain and shutdown the node")
```

Comment:
why is this changed? I don't see any corresponding change to the logic

Reply:
It was a typo on my side,the latest commit should resolve this.
Flag skip upgrade was changed to Int64 as ctx.Blockheight() is of type int64

Reply:
Yes, I agree with this change, and the rest of the diff looks quite clean.